### PR TITLE
Disjoint dataset for multi gpus & change learning rate schedule

### PR DESCRIPTION
**Pull request by gyeongin**

### CODE REVIEW

File: magenta/magenta/models/nsynth/ours/train.py [hunk 1]
```diff
@@ -127,7 +130,7 @@ def main(unused_argv=None):
         global_step=global_step,
         log_every_n_steps=FLAGS.log_period,
         local_init_op=local_init_op,
-        save_interval_secs=300,
+        save_interval_secs=1200,
```
Comment (jsjason) [inline magenta/magenta/models/nsynth/ours/train.py]: If we're going to change this, then how about making it info a command line argument while we're at it?

File: magenta/magenta/models/nsynth/ours/train.py [hunk 2]
```diff
@@ -55,6 +56,8 @@ def main(unused_argv=None):
 
   logdir = FLAGS.logdir
   tf.logging.info("Saving to %s" % logdir)
+  train_files = glob.glob(FLAGS.train_path + "/*")
```
Comment (jsjason) [inline magenta/magenta/models/nsynth/ours/train.py:61]: Have you checked that this works even if FLAGS.train_path ends with a slash character?

**Review by jsjason — COMMENTED** [review]
Nitpicking..

File: magenta/magenta/models/nsynth/ours/train.py (hunk 2 above)
Comment (gyeongin) [reply magenta/magenta/models/nsynth/ours/train.py:61]: Yes. Python `glob` handles both `....../dir//*` and `....../dir/*`.

File: magenta/magenta/models/nsynth/ours/train.py (hunk 1 above)
Comment (gyeongin) [reply magenta/magenta/models/nsynth/ours/train.py]: 해드릴게여

**Review by jsjason — APPROVED** [review]

**Pull request merged**